practice-one/src/components/Button/Button.css

```diff
@@ -0,0 +1,78 @@
+.btn {
+  border: none;
+  cursor: pointer;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 0 10px;
+  width: var(--btn-width, auto);
+  height: var(--btn-height, auto);
+}
+
+.align-default {
```

Comment:
move to style common file

Reply:
Because these two buttons are aligned to the left instead of centered like the other buttons, I split them up like that.
![image](https://github.com/datdragon777/React-Training/assets/74806826/e1b9d637-a0ff-4c59-a01c-6229b10007e9)

Reply:
yes, I know that, But this style it'll reuse another page, so need to create another style common to re-use